Code/Editor/MainWindow.cpp

```diff
@@ -942,20 +942,22 @@ void MainWindow::InitActions()
         .Connect(&QAction::triggered, this, &MainWindow::OnRefreshAudioSystem);
 
     // Game actions
-    am->AddAction(ID_VIEW_SWITCHTOGAME, tr("Play &Game"))
+    am->AddAction(ID_VIEW_SWITCHTOGAME, tr("Play Game"))
         .SetIcon(QIcon(":/stylesheet/img/UI20/toolbar/Play.svg"))
+        .SetToolTip(tr("Play Game"))
+        .SetStatusTip(tr("Activate the game input mode"))
+        .RegisterUpdateCallback(cryEdit, &CCryEditApp::OnUpdatePlayGame);
+    am->AddAction(ID_VIEW_SWITCHTOGAME_WINDOWED, tr("Play Game"))
         .SetShortcut(tr("Ctrl+G"))
         .SetToolTip(tr("Play Game (Ctrl+G)"))
         .SetStatusTip(tr("Activate the game input mode"))
-        .SetApplyHoverEffect()
         .SetCheckable(true)
         .RegisterUpdateCallback(cryEdit, &CCryEditApp::OnUpdatePlayGame);
-    am->AddAction(ID_VIEW_SWITCHTOGAME_FULLSCREEN, tr("Play &Game (Maximized)"))
+    am->AddAction(ID_VIEW_SWITCHTOGAME_FULLSCREEN, tr("Play Game (Maximized)"))
         .SetShortcut(tr("Ctrl+Shift+G"))
         .SetStatusTip(tr("Activate the game input mode (maximized)"))
-        .SetIcon(Style::icon("Play"))
-        .SetApplyHoverEffect()
-        .SetCheckable(true);
+        .SetCheckable(true)
+        .RegisterUpdateCallback(cryEdit, &CCryEditApp::OnUpdatePlayGame);
```

Comment:
I thought the same thing: it feels like pressing Ctrl-G should enter game mode in whatever is the current mode.

As for two ways of changing the setting being confusing, maybe just removing the settings toggle from the prefs screen, so that the dropdown menu is the only place to change it would help.